R/load.R

```diff
@@ -389,11 +531,28 @@ dotList <- function(input, spatial) {
       par.trigger <- TRUE
     }
     recipient <- list(
-      neighbours = unique(c(auxA$B, auxB$A, names(parallel)[!is.na(parallel)])),
-      before     = if (nrow(auxB) == 0) { NULL  } else { unique(auxB$A) },
-      after      = if (nrow(auxA) == 0) { NULL  } else { unique(auxA$B) },
-      parallel   = if ( !par.trigger  ) { NULL  } else { unique(names(parallel)[!is.na(parallel)]) },
-      edge       = if (nrow(auxA) == 0) { FALSE } else { any(auxA$Edge) })
+      neighbours = unique(c(auxA$B, auxB$A, 
+                            names(parallel)[!is.na(parallel)])),
+      before = if (nrow(auxB) == 0) { 
+        NULL  
+        } else { 
+          unique(auxB$A) 
+          },
+      after = if (nrow(auxA) == 0) { 
+        NULL  
+        } else { 
+          unique(auxA$B) 
+          },
+      parallel = if ( !par.trigger  ) { 
+        NULL  
+        } else { 
+          unique(names(parallel)[!is.na(parallel)]) 
+          },
+      edge = if (nrow(auxA) == 0) { 
+        FALSE 
+        } else { 
+          any(auxA$Edge) 
+          })
```

Comment:
```suggestion
      neighbours = unique(c(auxA$B, auxB$A, names(parallel)[!is.na(parallel)])),
      before     = ifelse(nrow(auxB) == 0, NULL, unique(auxB$A)),
      after      = ifelse(nrow(auxA) == 0, NULL, unique(auxA$B)),
      parallel   = ifelse(!par.trigger, NULL, 
                          unique(names(parallel)[!is.na(parallel)])),
      edge       = ifelse(nrow(auxA) == 0, FALSE, any(auxA$Edge)))
```

only the parallel line crossed the 80c limit here.